chia/consensus/multiprocess_validation.py

```diff
@@ -254,10 +254,11 @@ async def pre_validate_blocks_multiprocessing(
                 blockchain,
                 required_iters,
                 block,
-                sub_slot_iters=sub_slot_iters,
-                prev_ses_block=prev_ses_block,
+                sub_slot_iters=vs.current_ssi,
+                prev_ses_block=vs.prev_ses_block,
             )
         except ValueError:
+            log.exception("block_to_block_record()")
```

Comment:
Is this intentional? and if so, can we provide more information, or is mentioning `block_to_block_record()` enough here?

Reply:
I added it while debugging, but I think it makes sense to leave in. the `exception()` function prints a lot of relevant information about the exception already. I don't think there's anything else that should be added to that message.